### PR TITLE
Faster pandas build

### PR DESCRIPTION
**Pull request by lopuhin**
Now the tox step is taking about 110 s instead of 718 s. Pandas has wheels available for all python versions we need, it was just not picked up without the wheel package it seems.

So the builds are back to 3-4 minutes with this PR.

### CODE REVIEW

**Comment by kmike** [comment]
Weird, I was assuming wheel package is only needed to build wheels, not to use them. Thanks for the fix!

**Comment by lopuhin** [comment]
> Weird, I was assuming wheel package is only needed to build wheels, not to use them. Thanks for the fix!
hmm, actually ``wheel`` is not required - I just checked it with pandas locally, maybe there is something fishy here, let me double-check

**Comment by lopuhin** [comment]
Yeah, the change in this branch is not related to wheel speedup: https://travis-ci.org/TeamHG-Memex/eli5/builds/229393979?utm_source=github_status&utm_medium=notification

Probably the reason for speedup is that pandas wheels were uploaded yesterday (maybe late in the evening?): https://pypi.python.org/pypi/pandas/0.20.1
